core/include/detray/geometry/shapes/annulus2D.hpp

```diff
@@ -118,7 +118,7 @@ class annulus2D {
         const scalar_t minR_tol = bounds[e_min_r] - tol;
         const scalar_t maxR_tol = bounds[e_max_r] + tol;
 
-        assert(!detail::any_of(minR_tol >= scalar_t(0.f)));
+        assert(detail::any_of(minR_tol >= scalar_t(0.f)));
```

Comment:
Thanks for fixing this! I think it should actually be either
``` 
assert(detail::all_of(minR_tol >= scalar_t(0.f)));
```
or
```
assert(!detail::any_of(minR_tol < scalar_t(0.f)));
```

Reply:
Either one is fine by me :) Which one do you prefer? Thanks.

Reply:
I don't mind either :) the first one seems maybe a bit easier to read imo

Reply:
OK, I will fix it shortly. Thanks

Reply:
I'm going to propose patches to a few other unit tests in a separate PR a bit later too

Reply:
Changed in the new revision